src/java.desktop/macosx/classes/com/apple/laf/AquaLookAndFeel.java

```diff
@@ -327,7 +327,9 @@ public Object createValue(UIDefaults defaultsTable) {
 
         final ColorUIResource selectedTabTitlePressedColor = new ColorUIResource(240, 240, 240);
         final ColorUIResource selectedTabTitleDisabledColor = new ColorUIResource(new Color(1, 1, 1, 0.55f));
-        final ColorUIResource selectedTabTitleNormalColor = white;
+        final ColorUIResource selectedTabTitleNormalColor =
+                        System.getProperty("os.version").contains("10.16") ?
+                                    new ColorUIResource(Color.lightGray) : white;
```

Comment:
Can we read this information from the native? Probably it is already accessed by the SystemColor?

Reply:
What "information" you want to be read from native?

Reply:
The color instead of hardcoded "Color.lightGray"

Reply:
Well, even "white" was hardcoded till now...so I am not sure how can we read the color from native...do you have any idea?
Even CsystemColors.m has all the color hardcoded 
 sColors[java_awt_SystemColor_ACTIVE_CAPTION_TEXT] =        [NSColor blackColor];

Reply:
I dont think SystemColors are used in macos. I see in loadSystemColors() is commented in AquaLookAndFeel.java, therefore
loadNativeColors() defined in LWCToolkit.m is not used.

Reply:
> Well, even "white" was hardcoded till now...so I am not sure how can we read the color from native...do you have any idea?

The usage of the hardcoded white color was a bug, and it was found by the JDK-8251377. It would be good to investigate how to read the color of the text in the JTabbedPane(Or probably that color is used in some other components as well) from the native. 

> I dont think SystemColors are used in macos. I see in loadSystemColors() is commented in AquaLookAndFeel.java, therefore loadNativeColors() defined in LWCToolkit.m is not used.

SystemColors/LWCToolkit#loadNativeColors are for sure works on macOS it is responsible for the text selection color for example.

> Even CsystemColors.m has all the color hardcoded
> sColors[java_awt_SystemColor_ACTIVE_CAPTION_TEXT] = [NSColor blackColor];

All that hardcoded colors are bugs, but some of them use non-hardcoded values like: textColor/disabledControlTextColor/controlShadowColor/controlBackgroundColor/etc

Reply:
> SystemColors/LWCToolkit#loadNativeColors are for sure works on macOS it is responsible for the text selection color for example.
I could not find how loadNativeColors() is used/works because only place I see it is called in macos is in loadSystemColors() which is commented out in AquaLookAndFeel.